### PR TITLE
Us3 c cocktail card

### PR DESCRIPTION
**Pull request by OulG**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,5 +17,8 @@
   "devDependencies": {
     "concurrently": "^7.0.0",
     "husky": "^7.0.0"
+  },
+  "dependencies": {
+    "react-router-dom": "^6.3.0"
```
Comment (lchastanet) [inline package.json:22]: It shouldn't be here !

**Review by lchastanet — CHANGES_REQUESTED** [review]
Just remove the package in `package.json`

File: package.json (hunk 1 above)
Comment (OulG) [reply package.json]: I remove the package in package.json as you ask.

**Review by lchastanet — APPROVED** [review]
Ok nickel !

**Pull request merged**